### PR TITLE
Chore: TIP - Patch 8

### PR DESCRIPTION
**Pull request by ZarishIqbal**
### This PR contains the following changes:
- [x] Single Student Performance in the Comparison matrix - Ungraded SLOs are greyed out without a percentage
- [x] Student Learning Progress - Remedial-wise percentage tabs under each subject are clickable and take users to the `Skill View page` in Summative Results

### CODE REVIEW

File: web/src/pages/targeted-instruction/head-teacher-dashboard/tests/summative-test/index.tsx [hunk 1]
```diff
@@ -220,7 +221,14 @@ const HTSummative = () => {
 										gradeResult?.[subject]?.obt,
 										gradeResult?.[subject]?.total
 									)
-									return <PercentageBox key={subject} percentage={percentage} />
+
+									return (
+										<Link
+											key={subject + level}
+											to={`/targeted-instruction/summative-result/Level%20${level}/${subject}/result`}>
```
Comment (mudssrali) [inline web/src/pages/targeted-instruction/head-teacher-dashboard/tests/summative-test/index.tsx:228]: why we're using  `%2` ? adding space here?

**Review by mudssrali — APPROVED** [review]
Thanks @ZarishIqbal  Looks good to me.

File: web/src/pages/targeted-instruction/head-teacher-dashboard/tests/summative-test/index.tsx (hunk 1 above)
Comment (ZarishIqbal) [reply web/src/pages/targeted-instruction/head-teacher-dashboard/tests/summative-test/index.tsx:228]: It's actually %20.. level mein we're getting Kg, 1, 2, 3. 

We're supposed to pass Level KG, Level 1 and so on. 

%20 is the space we need.

Comment (mudssrali) [reply web/src/pages/targeted-instruction/head-teacher-dashboard/tests/summative-test/index.tsx:228]: at some point, we should get rid of these spaces and sanitize url

**Pull request merged**